tests/objcthemis/objthemis/SecureCellTestsSwift.swift

```diff
@@ -210,6 +210,188 @@ class SecureCellSealSwift: XCTestCase {
     }
 }
 
+// MARK: - Seal Mode (passphrase)
+
+class SecureCellSealPassphraseSwift: XCTestCase {
+
+    func testInitWithFixed() {
+        let cell = TSCellSeal(passphrase: "secret")
+        XCTAssertNotNil(cell)
+    }
+
+    func testInitWithRawData() {
+        let cell = TSCellSeal(passphraseData: "secret".data(using: .ascii)!)
+        XCTAssertNotNil(cell)
+    }
+
+    func testInitWithEmpty() {
+        XCTAssertNil(TSCellSeal(passphrase: ""))
+        XCTAssertNil(TSCellSeal(passphraseData: Data()))
+    }
+
+    func testRoundtrip() {
+        let cell = TSCellSeal(passphrase: "secret")!
+        let message = "All your base are belong to us!"
+        let context = "For great justice"
+
+        let encrypted = try? cell.encrypt(message.data(using: .utf8)!,
+                                          context: context.data(using: .utf8)!)
+        XCTAssertNotNil(encrypted)
+
+        let decrypted = try? cell.decrypt(encrypted!,
+                                          context: context.data(using: .utf8)!)
+        XCTAssertNotNil(decrypted)
+
+        let decryptedMessage = String(data: decrypted!, encoding: .utf8)
+        XCTAssertNotNil(decryptedMessage)
+
+        XCTAssertEqual(decryptedMessage, message)
+    }
+
+    func testDataLengthExtension() {
+        let cell = TSCellSeal(passphrase: "secret")!
+        let message = "All your base are belong to us!".data(using: .utf8)!
+
+        let encrypted = try! cell.encrypt(message)
+
+        XCTAssertGreaterThan(encrypted.count, message.count)
+    }
+
+    func testContextInclusion() {
+        let cell = TSCellSeal(passphrase: "secret")!
+        let message = "All your base are belong to us!".data(using: .utf8)!
+        let shortContext = ".".data(using: .utf8)!
+        let longContext = "You have no chance to survive make your time. Ha ha ha ha ...".data(using: .utf8)!
```

Comment:
O_o

Reply:
It makes sense... _[in context](https://en.wikiquote.org/wiki/Zero_Wing):_

> Operator: Main screen turn on.
>
> Captain: It's you *!!*
>
> CATS: How are you gentlemen *!!*
>
> CATS: All your base are belong to us.
>
> CATS: You are on the way to destruction.
>
> Captain: What you say *!!*
>
> CATS: You have no chance to survive make your time.
>
> CATS: Ha ha ha ha ....
>
> Operator: Captain *!!*